test/admin/dev.test.js

```diff
@@ -39,37 +47,37 @@ describe('Dev Function', () => {
   let apps;
   let app;
   let user;
-  let dev;
+  let client;
 
   beforeAll(() => test.setup());
   afterAll(() => test.teardown());
 
   beforeEach(async () => {
-    let adminClient = await getAuthenticatedClient(test.userData.apiKey.key);
+    const adminClient = await getAuthenticatedClient(test.userData.apiKey.key);
     test.groupId = test.userData.group.groupId;
     apps = await adminClient.apps(test.groupId);
     app = await apps.create({ name: 'testname' });
-    dev = adminClient.apps(test.groupId).app(app._id).dev();
 
     await createTestFunction(adminClient.apps(test.groupId).app(app._id).functions());
     await createUserPassProvider(adminClient.apps(test.groupId).app(app._id).authProviders());
     user = await createAppUser(adminClient.apps(test.groupId).app(app._id).users());
+
+    client = await createStitchClient(app.client_app_id);
   });
 
   afterEach(async () => {
     await apps.app(app._id).remove();
   });
 
   it('Supports executing the function', async () => {
-    const result = await dev.executeFunction(
-      user._id,
+    const result = await client.executeFunction(
       FUNC_NAME,
       777,
       23,
     );
 
-    expect(result.result.sum).toEqual({'$numberDouble': '800'});
-    expect(result.result.userId).toEqual(user._id);
+    expect(result.sum).toEqual(800);
```

Comment:
[ ⚠️  ]: calling out this test assertion change

Reply:
800 because of decoding extended json?

Reply:
@edaniels ... I'm not entirely sure why I needed to change this, but simply made it based off what I was seeing come in/out from the Stitch backend.

Reply:
Cool

Reply:
just to tie off loose ends, this change was _needed_ (read: not actually needed) b/c I inadvertently switched the test from using the admin api to the client api.  code has been reverted to use the proper api in #84